lib/file.coffee

```diff
@@ -43,6 +40,12 @@ class File
   onDidStatusChange: (callback) ->
     @emitter.on('did-status-change', callback)
 
+  loadRealPath: ->
```

Comment:
Why was this changed to use a promise? It doesn't seem like the promise is used anywhere, apologies if I'm missing something.